api/post.go

```diff
@@ -233,7 +235,8 @@ func getPosts(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	etagChan := app.Srv.Store.Post().GetEtag(id, true)
 
-	if !HasPermissionToChannelContext(c, id, model.PERMISSION_CREATE_POST) {
+	if !app.SessionHasPermissionToChannel(c.Session, id, model.PERMISSION_EDIT_POST) {
```

Comment:
The permission used here has changed. Is this intentional, or should this (and the line below) still be `PERMISSION_CREATE_POST`?